Extensions/TweenBehavior/JsExtension.js

```diff
@@ -0,0 +1,532 @@
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .addParameter("expression", _("To X"), "", false)
+      .addParameter("stringWithSelector", _("Easing"), easingChoices, false)
+      .setDefaultValue("linear")
+      .addParameter("expression", _("Duration"), "", false)
+      .addParameter(
+        "yesorno",
+        _("Destroy this object when tween finishes"),
+        "",
+        false
+      )
+      .setDefaultValue("no")
+      .getCodeExtraInformation()
+      .setFunctionName("addObjectPositionXTween");
+
+    behavior
+      .addAction(
+        "AddObjectPositionYTween",
+        _("Add object position Y tween"),
+        _("Add a tween animation for the object Y position."),
+        _(
+          "Tween the Y position of _PARAM0_ to _PARAM3_ with easing _PARAM4_ over _PARAM5_ms as _PARAM2_"
+        ),
+        _("Position"),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .addParameter("expression", _("To Y"), "", false)
+      .addParameter("stringWithSelector", _("Easing"), easingChoices, false)
+      .setDefaultValue("linear")
+      .addParameter("expression", _("Duration"), "", false)
+      .addParameter(
+        "yesorno",
+        _("Destroy this object when tween finishes"),
+        "",
+        false
+      )
+      .setDefaultValue("no")
+      .getCodeExtraInformation()
+      .setFunctionName("addObjectPositionYTween");
+
+    behavior
+      .addAction(
+        "AddObjectAngleTween",
+        _("Add object angle tween"),
+        _("Add a tween animation for the object angle."),
+        _(
+          "Tween the angle of _PARAM0_ to _PARAM3_° with easing _PARAM4_ over _PARAM5_ms as _PARAM2_"
+        ),
+        _("Angle"),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .addParameter("expression", _("To angle (in degrees)"), "", false)
+      .addParameter("stringWithSelector", _("Easing"), easingChoices, false)
+      .setDefaultValue("linear")
+      .addParameter("expression", _("Duration"), "", false)
+      .addParameter(
+        "yesorno",
+        _("Destroy this object when tween finishes"),
+        "",
+        false
+      )
+      .setDefaultValue("no")
+      .getCodeExtraInformation()
+      .setFunctionName("addObjectAngleTween");
+
+    behavior
+      .addAction(
+        "AddObjectScaleTween",
+        _("Add object scale tween"),
+        _(
+          "Add a tween animation for the object scale. (Note: the scale can never be less than 0)"
```

Comment:
I think period must go after the parenthesis, and no capital in parenthesis. See https://english.stackexchange.com/questions/6632/where-does-the-period-go-when-using-parentheses

`Add a tween animation for the object scale (note: the scale can never be less than 0).`